### PR TITLE
Lang and file icons

### PR DESCRIPTION
**Pull request by Matyrobbrt**
1. I have switched the theme class from **ThemeSettings** to **Theme**. I did that in order to be able to provide default themes and make it so that each theme can customize stuff like button bg colours.
2. Lang files (they go in _resources/lang/_ **and have to be cached before a new project is made**). in order to get a lang entry you can call **LangProvider.fromLang(translationKey)**
3. File icons can be put in _resources/icons/_ and have to have their name the same as the extension the icon will be used for (gradle, txt etc) (atm it only supports .png cuz for some reason it doesnt want to render a .svg)

### CODE REVIEW

File: src/main/java/io/github/railroad/utility/ImageUtility.java [hunk 1]
```diff
@@ -6,43 +6,108 @@
...
+				} catch (IOException e) {
+					e.printStackTrace();
+				} finally {
+					try {
+						br.close();
+					} catch (IOException e) {
+						e.printStackTrace();
+					}
+				}
+			} else**/
```
Comment (DaRealTurtyWurty) [inline src/main/java/io/github/railroad/utility/ImageUtility.java]: Is there a reason that this needs to be here?

Comment (Matyrobbrt) [reply src/main/java/io/github/railroad/utility/ImageUtility.java]: no. it is a total trash

Comment (DaRealTurtyWurty) [reply src/main/java/io/github/railroad/utility/ImageUtility.java]: Can't it be removed then?

Comment (Matyrobbrt) [reply src/main/java/io/github/railroad/utility/ImageUtility.java]: i removed it

**Review by DaRealTurtyWurty — APPROVED** [review]

**Pull request merged**